content/en/roadmap.md

```diff
@@ -9,79 +9,123 @@ type: page
 # Flux Roadmap
 
 {{% alert color="info" title="Production readiness" %}}
-The Flux custom resource definitions which are at `v1beta1`, `v1beta2` and `v2beta1`
-and their controllers are considered stable and production ready.
-Going forward, breaking changes to the beta CRDs will be accompanied by a conversion mechanism.
-Please see the [Migration and Support Timetable](flux/migration/timetable.md) for our commitment to end users.
-{{% /alert %}}
+The Flux latest [beta and stable APIs](https://fluxcd.io/flux/components/)
+(including their reconcilers) are well tested and safe to use in production environments.
 
-The following components are considered production ready:
+The beta APIs will be replaced in time with stable versions. Users have a six months window
+to upgrade from beta to stable, after an API reaches GA.
+{{% /alert %}}
 
-- [source-controller](/flux/components/source)
-- [kustomize-controller](/flux/components/kustomize)
-- [notification-controller](/flux/components/notification)
-- [helm-controller](/flux/components/helm)
-- [image-reflector-controller](/flux/components/image)
-- [image-automation-controller](/flux/components/image)
+## Milestones
 
-The following GitOps Toolkit APIs are considered production ready:
+The Flux team's current focus is to finalise the tasks from the [Flux GitOps GA](#flux-gitops-ga) milestone.
+We estimate that the Flux features part of this milestone will become general available in the first quarter of 2023.
```

Comment:
Maybe
```suggestion
We estimate that the Flux features part of this milestone will become generally available in the first quarter of 2023.
```
refer https://github.blog/changelog/2021-06-21-github-packages-container-registry-is-generally-available/ .